docs/joule.md

```diff
@@ -1,7 +1,7 @@
-Intel Joule {#grossetete}
+Intel Joule
```

Comment:
I think this one needs `{#joule}`, just like the old version had `{#grossetete}`, as it's used as the anchor by the docs generator and it's a corresponding "endpoint" for `@ref joule` in `index.md`